supervisor/docker/interface.py

```diff
@@ -99,6 +99,8 @@ def _install(
                 docker_image.tag(image, tag="latest")
         except docker.errors.APIError as err:
             _LOGGER.error("Can't install %s:%s -> %s.", image, tag, err)
+            free_space = self.coresys.host.info.get_disk_free_space()
```

Comment:
We could check `err.response.status_code == 404:` here and add some additional message in the log.

e.g., `This error is often caused by not having enough disk space available.`